tokens/connect/coinbase-wallet.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Coinbase Wallet
+description: This guide walks you through how to use the Coinbase Wallet extension and the Mobile App.
+---
+
+# Interacting with Moonbeam Using the Coinbase Wallet Mobile App  
+
+![Coinbase Wallet Banner](/images/tokens/connect/coinbase-wallet/coinbase-wallet-banner.png)
+
+## Introduction {: #introduction } 
+
+[Coinbase Wallet](https://wallet.coinbase.com/?_branch_match_id=977295450874474909&_branch_referrer=H4sIAAAAAAAAA8soKSkottLXT8%2FXS07SLddLzs%2FVD8%2FJynFKSy02zE4CAFZ0JzQfAAAA){target=_blank} is a self-custody (non-custodial) wallet like [MetaMask](/tokens/connect/metamask/) available as a mobile application for iOS and Android. There is also a browser extension version of Coinbase Wallet, but the extension does not support the addition of custom networks and is thus incompatible with Moonbeam at this time. Coinbase Wallet is an entirely different product from Coinbase Exchange, a custodial platform for buying and selling cryptocurrency. You can interact with Moonbeam, Moonriver, and the Moonbase Alpha testnet in Coinbase Wallet after adding them as custom networks.
+
+!!! note
+    You can add custom tokens to your Coinbase Wallet by importing the contract address. However, these custom tokens may not be supported by Coinbase Exchange. If you send a custom token from your Coinbase Wallet to Coinbase Exchange that is not supported by Coinbase Exchange, you will lose those funds forever.  
+
+In this guide, we'll go through the process of setting up the Coinbase Wallet mobile application and configuring it for the Moonbeam Network. 
+
+## Install the Coinbase Wallet App {: #install-the-coinbase-wallet-app } 
+
+You can download the Coinbase Wallet app from the iOS App Store or the Google Play Store. Coinbase maintains [this homepage](https://wallet.coinbase.com/?_branch_match_id=977295450874474909&_branch_referrer=H4sIAAAAAAAAA8soKSkottLXT8%2FXS07SLddLzs%2FVD8%2FJynFKSy02zE4CAFZ0JzQfAAAA){target=_blank} with the download links for both.
```

Comment:
I prefer to use more descriptive link text, I think what you could do here is something like

```You can [download the Coinbase Wallet app](https://wallet.coinbase.com/?_branch_match_id=977295450874474909&_branch_referrer=H4sIAAAAAAAAA8soKSkottLXT8%2FXS07SLddLzs%2FVD8%2FJynFKSy02zE4CAFZ0JzQfAAAA){target=_blank} from the iOS App Store or the Google Play Store.```